src/main/java/wooteco/subway/domain/path/DiscountPolicy.java

```diff
@@ -0,0 +1,52 @@
+package wooteco.subway.domain.path;
+
+import static wooteco.subway.domain.path.DiscountPolicy.Constants.*;
+
+import java.util.Arrays;
+import java.util.function.Function;
+import java.util.function.Predicate;
+
+enum DiscountPolicy {
```

Comment:
이름은 할인정책인데 구체적으로는 나이 구간에 대한 내용이네요

Reply:
연령에 관한 할인 정책이라는 의미를 담아 개명해보았습니다. 감사합니다!
```suggestion
enum AgeDiscountPolicy {
```